spec/models/book_author_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
```

Comment:
Let's make sure that this is the right naming convention when we have our check-ins tomorrow

Reply:
Let's open an issue to track it so we remember to ask about it on Thursday.

Reply:
...or tomorrow.